app/CocoaPods/CPUserProject.m

```diff
@@ -241,6 +241,11 @@ - (void)taskDidFinish;
                                                   name:NSFileHandleDataAvailableNotification
                                                 object:nil];
 
+  NSUserNotification *completionNotification = [[NSUserNotification alloc] init];
+  completionNotification.title = @"Pod Workspace Ready";
```

Comment:
Can you move this to a localised string please?

https://github.com/CocoaPods/CocoaPods-app/blob/master/app/CocoaPods/Supporting%20Files/Localizable.strings